src/main/scala/contracts/token_sale_setup_proxy/v1.0.0/ergoscript/Token_Sale_Setup_Proxy.es

```diff
@@ -69,11 +70,11 @@
             val validCollectionIssuerProperties: Boolean = {
 
                 allOf(Coll(
-                    (tokenCollectionIssuerBoxOUT.R4[Int].get == collectionStandardVersion),
-                    (tokenCollectionIssuerBoxOUT.R5[Coll[Coll[Byte]]].get == collectionInfo),
-                    (tokenCollectionIssuerBoxOUT.R6[Coll[(Coll[Byte], Coll[Byte])]].get == collectionSocials),
-                    (tokenCollectionIssuerBoxOUT.R7[Long].get == collectionMintingExpiry),
-                    (tokenCollectionIssuerBoxOUT.R8[Coll[(Coll[Byte], Coll[Byte])]].get == collectionAdditionalInfo)
+                    (tokenCollectionIssuerBoxOUT.R4[Int].get == CollectionStandardVersion),
+                    (tokenCollectionIssuerBoxOUT.R5[Coll[Coll[Byte]]].get == CollectionInfo),
+                    (tokenCollectionIssuerBoxOUT.R6[Coll[(Coll[Byte], Coll[Byte])]].get == CollectionSocials),
+                    (tokenCollectionIssuerBoxOUT.R7[Long].get == CollectionMintingExpiry),
+                    (tokenCollectionIssuerBoxOUT.R8[Coll[(Coll[Byte], Coll[Byte])]].get == CollectionAdditionalInfo)
```

Comment:
these values come from context vars, so they can be anything the tx operator wants